rmg.py

```diff
@@ -51,12 +51,11 @@ def parse_command_line_arguments(command_line_args=None):
     sensible, parses them, and returns them.
     """
 
-    parser = argparse.ArgumentParser(description="""
-                                                 Reaction Mechanism Generator (RMG) is an automatic chemical reaction
-                                                 mechanism generator that constructs kinetic models composed of
-                                                 elementary chemical reaction steps using a general understanding of
-                                                 how molecules react.
-                                                 """)
+    parser = argparse.ArgumentParser(description='Reaction Mechanism Generator (RMG) is an automatic chemical reaction '
+                                                 'mechanism generator that constructs kinetic models composed of '
+                                                 'elementary chemical reaction steps using a general understanding of '
+                                                 'how molecules react.')
```

Comment:
Interesting--so I checked that the message looked the same from the command line (by running python rmg.py -h) before and after the change and I thought it was displaying the same way. If not though, we will have to get clever because apparently the `ArgumentParser` function removes this from the output. For example, check out the message before and after changing the description to the following:

```
    parser = argparse.ArgumentParser(description='Reaction Mechanism Generator (RMG) is an automatic chemical reaction\n\n\n\n\n'
                                                 'mechanism generator that constructs kinetic models composed of\n'
                                                 'elementary chemical reaction steps using a general understanding of\n'
                                                 'how molecules react.')
```

Reply:
Oh, that's interesting! Maybe argparse automatically formats the description so it looks nice. I didn't check the behavior before commenting.

In that case, we can leave it.